### PR TITLE
Update MechJeb2.csproj

### PR DESCRIPTION
**Pull request by RCrockford**
Updated to current working state.

### CODE REVIEW

**Comment by sarbian** [comment]
DLL hints should be in your .csproj.user and not in the main .csproj
Removing the Dll would break the jenkins build
I can accept the file list changes and the C version update.

**Comment by RCrockford** [comment]
Hint paths should be back to normal, sorry about that.

File: MechJeb2/MechJeb2.csproj [hunk 1]
```diff
@@ -264,6 +233,7 @@
     <Compile Include="ScriptsModule\MechJebModuleScriptActionWhile.cs" />
     <Compile Include="ScriptsModule\MechJebModuleScriptActionsList.cs" />
     <Compile Include="ScriptsModule\MechJebModuleScriptCondition.cs" />
+    <Compile Include="Shepperd.cs" />
```
Comment (lamont-granquist) [inline MechJeb2/MechJeb2.csproj]: 👍

**Comment by sarbian** [comment]
Ok, let's merge is as is.

FTI the project has more dll reference than you have because on Linux and OSX Unity has split its engine dll in multiple one and for some reason they did not do it on WIndows until a release later...

**Pull request merged**